frontend/pages/hosts/details/cards/Activity/CommandFeed/CommandFeed.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+import { ICommand } from "interfaces/command";
+import { IGetCommandsResponse } from "services/entities/command";
+
+import Pagination from "components/Pagination";
+
+import EmptyFeed from "../EmptyFeed/EmptyFeed";
+import CommandItem, {
+  ShowCommandDetailsHandler,
+} from "../CommandItem/CommandItem";
+
+const baseClass = "command-feed";
+
+interface ICommandFeedProps {
+  commands: IGetCommandsResponse;
+  onShowDetails: ShowCommandDetailsHandler;
+  onNextPage: () => void;
+  onPreviousPage: () => void;
+}
+
+const CommandFeed = ({
+  commands,
+  onShowDetails,
+  onNextPage,
+  onPreviousPage,
+}: ICommandFeedProps) => {
+  const { meta, results } = commands;
+  if (results === null || results.length === 0) {
+    return (
+      <EmptyFeed
+        title="No MDM commands"
+        message="Completed MDM commands will appear here."
```

Comment:
```suggestion
        message="MDM commands will appear here."
```

Idea for a possible quick fix  ^